cleanlab/count.py

```diff
@@ -111,8 +103,8 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
        TL;DR: use this method to get the most accurate estimate of number of label issues when you don't need the indices of the label issues.
 
     multi_label : bool, optional
-       Refer to documentation for this argument in :py:func:`compute_confident_joint <cleanlab.count.compute_confident_joint>` with `multi_label=True` for details.
-
+      Whether dataset is for multi-label classification or multi-class classfication,
```

Comment:
Does both multi-label and multi-class classification use the `multi_label=True` boolean? Seems a little confusing to mention multi-class classification under the `multi_label` explanation.

Reply:
`multi_label=True` specifies that the ML task is multi-label not multi-class. Where did you see discussion of multi-class classification under `multi_label=True`?

Reply:
The logic of "Whether dataset is for multi-label classification or multi-class classification see link" and then link starting with "If True, this is multi-label classification rather than multi-class classification" was be a bit confusing. Might be clearer to say "If false, this is regular multi-class classification. If true, see link for details."

Reply:
updated everywhere to follow this structure